airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -99,7 +107,7 @@
         "properties": {
           "access_token": {
             "type": "string",
-            "path_in_connector_config": ["credentials", "access_token"]
+            "path_in_connector_config": ["credentials", "access_token", "api_url"]
```

Comment:
Have you tested that oAuth still works for this connector?

Reply:
Nope, unintended addition

Reply:
Removed and resolved